C#/C#.sublime-syntax

```diff
@@ -858,6 +860,10 @@ contexts:
         1: keyword.control.using.cs
         2: meta.group.cs punctuation.section.group.begin.cs
       set: [using_block, line_of_code]
+    - match: \b(using)\s*
```

Comment:
I don't know the context, but I presume it isn't possible for another identifier starting with `using` to be present here?

Reply:
I believe that is correct, yes. If one wants to define or reference a variable called `using`, it has to be escaped with `@` - `var @using = 123;`